docs/aws-copilot.md

```diff
@@ -1,14 +1,50 @@
-# AWS Copilot Ops Manual
+# WIP - AWS Copilot Ops Manual
 
-## Configure AWS local credentials
+This document aims to provide instructions on how to work with aws-copilot, and creating and managing the environments and services to get the MAVIS application up and running.
 
-Install `awscli` and `aws-copilot`. Then configure your AWS CLI credentials
-locally:
+## Prerequisites
+
+Before you start, make sure you have the following ready:
+
+1. Credentials to your AWS account.
+2. Administrative rights on your machine for software installation.
+
+### Installing AWS CLI
+
+The AWS Command Line Interface (CLI) is really useful for managing AWS services from the command line; you might require it for myriad of tasks, so having it at your disposal is essential.
+
+**MacOS/Linux/WSL Installation:**
+
+1. Open your Terminal application.
+2. Execute the following command to download and install AWS CLI version 2.
 
 ```bash
-$ aws configure sso
+curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+unzip awscliv2.zip
+sudo ./aws/install
 ```
 
+3. Confirm the installation by executing aws --version, which should display the installed version.
+
+### Installing AWS Copilot
+
+AWS Copilot simplifies deploying containerized applications on AWS, automating numerous steps in the deployment process.
+
+**MacOS/Linux/WSL Installation:**
+
+1. Open your command-line interface.
+2. Use the command below:
+
+```bash
+asdf plugin add aws-copilot
+asdf install aws-copilot 1.32.0
```

Comment:
The version is in the `.tool-versions` file, so I don't think it needs to be specified here.

```suggestion
asdf install aws-copilot
```